### ISSUE TITLE
kotest-bom: how to import to project dependencies?

### ISSUE THREAD

**Issue by glockbender** [kotest/kotest#2387]
I am trying to create my own BOM and import other bom-s into it. But when I try to import 'kotest-bom' it cannot be found with either gradle or maven. 

my BOM example:
```
            <dependency>
                <groupId>io.kotest</groupId>
                <artifactId>kotest-bom</artifactId>
                <version>${kotest.version}</version> // here I've used 4.6.1
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```
or my attempt to add bom to project:
```testImplementation(platform("io.kotest:kotest-bom:4.6.1"))```

gradle error: 

> Could not find artifact io.kotest:kotest-bom-jvm:pom:4.6.1 in central (https://repo.maven.apache.org/maven2)

Also, I can't find such an artifact in the repository.
Could you please publish `bom` in the repository or specify its `artifactId`?

**Comment by sksamuel** [comment]
I've published a BOM for 4.6.3 and it will be included going forward.
https://repo1.maven.org/maven2/io/kotest/kotest-bom/

**Comment by glockbender** [comment]
Thanks a lot
